deeprank/learn/NeuralNet.py

```diff
@@ -1135,7 +1150,7 @@ def _compute_hitrate(self, irmsd_thr=4.0):
                     binary_recomendation, npos)
                 self.data[l]['relevance'] = binary_recomendation
 
-    def _get_relevance(self, data, irmsd_thr=4.0):
+    def _get_relevance(self, data, target_thr=4.0):
```

Comment:
If we make target generic but not limited to IRMSD, we have to require users to set `target_thr`, so `4.0` should be removed.
Also it's needed to update line 853 that calls `_get_relevance`.